README.md

```diff
@@ -11,15 +11,19 @@ protobuf message as fast as possible.
 
 See [0x7f/protog-benchmark](https://github.com/0x7f/protog-benchmark) for benchmarks.
 
-## Build
-
-First, install required packages, e.g. on Debian run:
+## Install required packages
 
+Debian:
 ```
 apt-get install build-essential cmake protobuf-compiler libprotobuf-dev libprotoc-dev libyajl-dev
 ```
 
-Then build:
+Mac OS X with Homebrew:
+```
+brew install gcc cmake protobuf protobuf-c yajl
```

Comment:
Are you sure `protobuf-c` is required? This is the protobuf compiler that generates C code which is not used in this project.